test/stress/child.js

```diff
@@ -0,0 +1,78 @@
+const https = require('https')
+const path = require('path')
+const fs = require('fs')
+const uuid = require('uuid')
+const _ = require('lodash/fp')
+const { PerformanceObserver, performance } = require('perf_hooks')
+
+const utils = require('./utils')
+const variables = require('./utils/variables')
+
+function getCert (machineIndex) {
+  try {
+    return {
+      key: fs.readFileSync(path.resolve(__dirname, 'machines', `${machineIndex}`, 'client.key')),
+      cert: fs.readFileSync(path.resolve(__dirname, 'machines', `${machineIndex}`, 'client.pem'))
+    }
+  } catch (e) {
+    return null
+  }
+}
```

Comment:
better avoid blocking synchronous code whenever we can, try something like this instead:

```suggestion
const pify = require('pify')
const fs = pify(require('fs'))
function getCert (machineIndex) {
  return fs.readFile(...).then(file => ...).catch("you can also catch this somewhere else")
}
```

Reply:
My initial idea was really to keep the blocking going. This is all running as a single script, and I kind of need the machine creation to be predictable. I'll ask Rafael about it tho

Reply:
Just confirmed with Rafael, he believes it is worth the machine creation to be asynchronous. Working on it!